### PR TITLE
First draft of container docs. WIP

### PR DESCRIPTION
**Pull request by s1hofmann**

### CODE REVIEW

File: content/docs/enterprise/container.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+
+### Copy demo testcase
+ADD ./testsuite-a $HOME/sakuli_testsuite
+ENV SAKULI_TEST_SUITE=$HOME/sakuli_testsuite
+{{</highlight>}}
+
+### Configure what to execute on `npm test`
+
+The main configuration file of an npm project is its `package.json` file.
+Within this file it's possible to configure [npm-scripts](https://docs.npmjs.com/misc/scripts), a handy way to execute scripts inside an npm project.
```
Comment (YunaAmnell) [inline content/docs/enterprise/container.md]: "inside **a** npm project"

File: content/docs/enterprise/container.md [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+
+{{<highlight js>}}
+...
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+...
+{{</highlight>}}
+
+`npm test` is the default way of executing tests in an npm project, so Sakuli tests should be executed this way, too!
```
Comment (YunaAmnell) [inline content/docs/enterprise/container.md]: "in **a** npm project"

File: content/docs/enterprise/container.md [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+
+When running a Docker container it is possible to mount a file or directory on the Docker host into a container.
+This mechanism can be used to provide a Sakuli test to a test container:
+
+{{<highlight bash>}}
+docker run -v /path/to/test/project/on/host:/sakuli_test -e SAKULI_LICENSE_KEY=<YOUR SAKULI LICENSE KEY> taconsol/sakuli:2.1.2
+{{</highlight>}}
+
+By adding the **-v** parameter we're mounting the root folder of our Sakuli test at `/path/to/test/project/on/host` (where the `package.json` file is located) on our host machine at `/sakuli_test` inside our test container.
+Inside the test container we could now run a Sakuli test via `sakuli run /sakuli_test/test_suite_folder`.
```
Comment (cliffle) [inline content/docs/enterprise/container.md]:
```suggestion
Inside the test container we can now run a Sakuli test via `sakuli run /sakuli_test/test_suite_folder`.
```

File: content/docs/enterprise/container.md [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+
+Docker allows to pass environment variable along other parameters when starting a new container:
+
+{{<highlight bash>}}
+docker run --rm -p 5901:5901 -p 6901:6901 -e SAKULI_LICENSE_KEY=<YOUR SAKULI LICENSE KEY> taconsol/sakuli:2.1.2
+{{</highlight>}}
+
+Parameters:
+
+- **\-\-rm**: The test container is removed after execution, not just stopped
```
Comment (cliffle) [inline content/docs/enterprise/container.md]:
```suggestion
- **\-\-rm**: The test container will be removed after execution, not just stopped
```

**Pull request merged**